dev-python/kaitaistruct/kaitaistruct-0.8.ebuild

```diff
@@ -0,0 +1,16 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} pypy )
+
+inherit distutils-r1
+
+DESCRIPTION="Kaitai Struct runtime for Python"
+HOMEPAGE="https://kaitai.io/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```

Comment:
Missing dep on setuptools.

I suppose running tests is not an option here.

Reply:
Hello @mgorny 

I've removed the older ebuild; added the dependency on setuptools. As far as I understand https://github.com/kaitai-io/kaitai_struct the "tests" folder is another git repository and `ebuild kaitaistruct-0.8.ebuild fetch` fetches a tarball without it.

I also squashed all my commits into a single one.